jfrog-cli/artifactory/utils/golang/project/dependencies/dependenciesutils.go

```diff
@@ -0,0 +1,452 @@
+package dependencies
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/jfrog/jfrog-cli-go/jfrog-cli/artifactory/utils"
+	"github.com/jfrog/jfrog-cli-go/jfrog-cli/artifactory/utils/golang"
+	"github.com/jfrog/jfrog-cli-go/jfrog-cli/utils/config"
+	"github.com/jfrog/jfrog-client-go/artifactory/buildinfo"
+	"github.com/jfrog/jfrog-client-go/httpclient"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	multifilereader "github.com/jfrog/jfrog-client-go/utils/io"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils/checksum"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"unicode"
+)
+
+// Collects the dependencies of the project
+func CollectProjectNeededDependencies(targetRepo string, cache *golang.DynamicCache,details *config.ArtifactoryDetails) (map[string]bool, error) {
```

Comment:
I suggest renaming the function to *CollectDependencies*.

Reply:
Changed to CollectProjectDependencies